--- eza-0.14.2/Cargo.toml	1970-01-01T00:00:01+00:00
+++ eza-0.14.2/Cargo.toml	2023-10-17T11:29:30.319488+00:00
@@ -20,7 +20,14 @@
     "/Justfile",
     "/Vagrantfile",
     "/screenshots.png",
+    "/snap",
     "/tests",
+    "/cliff.toml",
+    "/deb.asc",
+    "/deny.toml",
+    "/flake.*",
+    "/rust-toolchain.toml",
+    "/treefmt.nix",
 ]
 description = "A modern replacement for ls"
 homepage = "https://github.com/eza-community/eza"
@@ -90,10 +97,6 @@
 [[bin]]
 name = "eza"
 
-[[bench]]
-name = "my_benchmark"
-harness = false
-
 [dependencies.ansiterm]
 version = "0.12.2"
 
@@ -157,10 +160,6 @@
 [dependencies.zoneinfo_compiled]
 version = "0.5.1"
 
-[dev-dependencies.criterion]
-version = "0.5.1"
-features = ["html_reports"]
-
 [dev-dependencies.trycmd]
 version = "0.14"
 
@@ -172,14 +171,10 @@
 [features]
 default = ["git"]
 git = ["git2"]
-nix = []
-nix-local = []
 vendored-libgit2 = ["git2/vendored-libgit2"]
-vendored-openssl = ["git2/vendored-openssl"]
 
 [target."cfg(target_os = \"linux\")".dependencies.proc-mounts]
 version = "0.3"
 
 [target."cfg(unix)".dependencies.uzers]
 version = "0.11.3"
-
